[2022-11-27T09:44:07.589+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2022-11-27T09:43:59.762536+00:00 [queued]>
[2022-11-27T09:44:07.605+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2022-11-27T09:43:59.762536+00:00 [queued]>
[2022-11-27T09:44:07.607+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T09:44:07.608+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-27T09:44:07.610+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T09:44:07.628+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2022-11-27 09:43:59.762536+00:00
[2022-11-27T09:44:07.645+0000] {standard_task_runner.py:55} INFO - Started process 4695 to run task
[2022-11-27T09:44:07.654+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2022-11-27T09:43:59.762536+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp9usjxzod']
[2022-11-27T09:44:07.666+0000] {standard_task_runner.py:83} INFO - Job 14: Subtask is_api_available
[2022-11-27T09:44:07.773+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2022-11-27T09:43:59.762536+00:00 [running]> on host c7f7231a8603
[2022-11-27T09:44:07.871+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2022-11-27T09:43:59.762536+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-27T09:43:59.762536+00:00
[2022-11-27T09:44:07.875+0000] {http.py:120} INFO - Poking: api/
[2022-11-27T09:44:07.901+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2022-11-27T09:44:07.914+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2022-11-27T09:44:08.371+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2022-11-27T09:44:08.387+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20221127T094359, start_date=20221127T094407, end_date=20221127T094408
[2022-11-27T09:44:08.434+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-27T09:44:08.487+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
